l1-contracts/deploy-scripts/PrepareZKChainRegistrationCalldata.s.sol

```diff
@@ -0,0 +1,286 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+// solhint-disable no-console
+
+import {Script, console2 as console} from "forge-std/Script.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+
+import {IBridgehub} from "contracts/bridgehub/IBridgehub.sol";
+import {Bridgehub} from "contracts/bridgehub/Bridgehub.sol";
+import {L2ContractHelper} from "contracts/common/libraries/L2ContractHelper.sol";
+import {AddressAliasHelper} from "contracts/vendor/AddressAliasHelper.sol";
+import {L1SharedBridge} from "contracts/bridge/L1SharedBridge.sol";
+import {IGovernance} from "contracts/governance/IGovernance.sol";
+import {Utils} from "./Utils.sol";
+
+contract PrepareZKChainRegistrationCalldataScript is Script {
+    using stdToml for string;
+
+    address internal constant ADDRESS_ONE = 0x0000000000000000000000000000000000000001;
+
+    struct Config {
+        address chainAdmin;
```

Comment:
we should add a link to the documentation where we say what chainAdmin can and cannot do. 
do we have something like this?

Reply:
I'm not aware of any, but I can add later if I stumble upon it

Reply:
And this is the adress on L1 or on L2? (as it doesn't have to be the EOA, right?)

Reply:
it's L1 address, which is not necessarily an EOA

Reply:
let's mention it (that this is L1 address)